docs/clearml_data/clearml_data_cli.md

```diff
@@ -32,9 +32,11 @@ Creates a new dataset. <br/>
 
 
 
-:::important
-clearml-data works in a stateful mode so once a new dataset is created, the following commands
+:::tip Task ID
+* clearml-data works in a stateful mode so once a new dataset is created, the following commands
 do not require the `--id` flag.
+* To locate a task ID, go to the task's info panel in the [WebApp](../webapp/webapp_overview.md). In the top of the panel, 
```

Comment:
@pollfly Does this addition make sense where the note discusses *not* needing the ID?

Reply:
Would it be better to put the admonition after the description of the `create` command, or after the general description of the CLI (right before the `Creating a Dataset` heading)?